### PR TITLE
fix the api example for create_global_var, create_parameter, SGDOptim…

### PR DESCRIPTION
**Pull request by wopeizl**
…izer, RMSPropOptimizer, MomentumOptimizer, LarsMomentumOptimizer, FtrlOptimizer

test=develop

sgdoptimizer

### CODE REVIEW

**Review by shanyi15 — DISMISSED** [review]

File: python/paddle/fluid/optimizer.py [hunk 1]
```diff
@@ -1746,8 +1837,31 @@ class FtrlOptimizer(Optimizer):
...
+
+                fetch_list = [avg_cost]
+                train_reader = paddle.batch(
+                    paddle.dataset.uci_housing.train(), batch_size=1)
+                feeder = fluid.DataFeeder(place=place, feed_list=[x, y])
+                exe = fluid.Executor(place)
+                exe.run(fluid.default_startup_program())
+                for data in train_reader():
+                    exe.run(main, feed=feeder.feed(data), fetch_list=fetch_list)
+                    break
```
Comment (chengduoZH) [inline python/paddle/fluid/optimizer.py]: Why add `break` here?

File: python/paddle/fluid/optimizer.py [hunk 2]
```diff
@@ -1610,8 +1678,31 @@ class RMSPropOptimizer(Optimizer):
...
```
Comment (chengduoZH) [inline python/paddle/fluid/optimizer.py]: Why add `break` here?

File: python/paddle/fluid/optimizer.py [hunk 3]
```diff
@@ -520,8 +520,31 @@ class SGDOptimizer(Optimizer):
...
 
-            sgd_optimizer = fluid.optimizer.SGD(learning_rate=0.2)
-            sgd_optimizer.minimize(cost)
+            import paddle
+            import paddle.fluid as fluid
+            import numpy as np
+
+            place = fluid.CPUPlace()
+            main = fluid.Program()
+            with fluid.program_guard(main):
```
Comment (chengduoZH) [inline python/paddle/fluid/optimizer.py:532]: Why use `fluid.program_guard` here?

File: python/paddle/fluid/optimizer.py (hunk 2 above)
Comment (wopeizl) [reply python/paddle/fluid/optimizer.py]: removed

File: python/paddle/fluid/optimizer.py (hunk 1 above)
Comment (wopeizl) [reply python/paddle/fluid/optimizer.py]: removed

**Review by shanyi15 — APPROVED** [review]

**Review by chengduoZH — APPROVED** [review]

**Pull request merged**